site/app/templates/NotificationSettings.twig

```diff
@@ -42,6 +42,44 @@
                 {% endif %}
             </div>
 
+            <div class="heading">
+                <h2>Registration</h2>
+                <hr>
+            </div>
+            <div class="option row">
+                <div class="option-desc col-md-4">
+                    <span>
+                        <span class="option-title">New Self Registration</span>
+                        <br>
+                        <span class="option-alt">Notify me of self-registrations</span>
```

Comment:
I'm still not entirely satisfied with the wording here, but I don't have a good alternative to suggest.  Deferring to @bmcutler for UI/wording review.